src/controllers/eventController.js

```diff
@@ -83,12 +93,15 @@ eventController.put('/:id', async (req, res) => {
         if (Object.keys(req.body).length === 0) {
             throw new Error('Invalid request Body!')
         }
+        if (req.body.dates[0].startTime >= req.body.dates[0].endTime) {
+            throw new Error('Start-time can\'t be after or equal to end-time!')
+        }
         
         const event = await findEventByID(req.params.id);
 
         if (
             req.requester?._id === undefined ||
-            (req.requester._id != event?.creator &&
+            (req.requester._id != event?.creator._id &&
```

Comment:
Again once loose equality and once strict equality?

Reply:
![image](https://github.com/Getsov/Motorsport-Events-Server/assets/50380455/770aafcd-aec0-4695-8f6f-195c4840b428)
![image](https://github.com/Getsov/Motorsport-Events-Server/assets/50380455/9f75e8d4-1875-4d43-b6f5-f66bdc4ff817)

In case if i am admin and did not use that check

Reply:
You are right, i dont need 102 row.

Reply:
I'm talking about the use of the two types of checks at once.

Reply:
fixed